surveys/templates/survey_submitted_detail.html

```diff
@@ -44,6 +96,72 @@ <h2 class="card-title">{{ form_entry.name }}</h2>
       "leftColumns": 1
     }
   });
+
+  var $totalForms = $('[name$=TOTAL_FORMS]');
+  var chartContainer = '[class^="chart-container-"]'
+
+  function resetChartOrder() {
```

Comment:
I'm not sure how difficult this is, but a moving chart animation would be really nice

Reply:
Very true. A cursory Google search didn't return an immediately obvious way to do this, so I punted to #125.